packages/tlschema/src/shapes/TLEmbedShape.ts

```diff
@@ -15,7 +15,7 @@ const safeParseUrl = (url: string) => {
 }
 
 /** @public */
-export const EMBED_DEFINITIONS = [
+export const DEFAULT_EMBED_DEFINITIONS = [
```

Comment:
Do we really really need our embed definitions in the schema? I never really liked this data being in the schema as opposed to being in the tldraw library. The consequences for having a bad name pass validation also seem pretty low, so long as we properly show "broken embeds" in the embed shape.

Reply:
I guess so. We don't actually do any validations on the type of the embed, since it is always resolved on the fly from the URL.

Schema does reference it once in a migration [here](https://github.com/tldraw/tldraw/blob/main/packages/tlschema/src/shapes/TLEmbedShape.ts#L686). And it seems that this is the main reason why these definitions live in the schema. 

One use case where it still might be useful to have access to the definitions is if one of the embeds changed their url. Then it could be easier to scope down the migration to use a new url. We could hard code it though 🤷‍♂️

Reply:
i think we can just pull out the `fromEmbedUrl` logic for each embed type that was supported before that migration was written and inline it in the migration? From then on I don't think we'd ever need to migrate these URLs, even if the apps change their format, because we can just put that logic in the `toEmbedUrl` methods?

Reply:
ah ignore me this might be more bother than it's worth.